spec/src/utils/request-queue.js

```diff
@@ -14,9 +14,10 @@ dotenv.config();
 
 describe('ConstructorIO - Utils - Request Queue', () => {
   const storageKey = '_constructorio_requests';
-  const waitInterval = 700;
+  const waitInterval = 1000;
```

Comment:
Awesome! For my own learning, can you explain how this fixes the issue with utils tests?

Reply:
It's actually the other line change that fixes the problem: 

```
trackingSendDelay: 1,
```

We recently changed the trackingSendDelay default from `150` (ms) to `250`. Since the test wait interval was 700, and some of the test required dispatching three events, it would sometimes take longer than the 700ms wait interval (250 * 3 = 750).

Changing the send delay to something very short ensures that we have plenty of time for the request queue to "clear" before checking the results. The change from `700` to `1000` was basically made just because I like "round" numbers 😛

Reply:
Thank you!